test/powershell/Modules/Microsoft.PowerShell.Utility/clixml.tests.ps1

```diff
@@ -218,6 +266,22 @@ Describe "Deserializing corrupted Cim classes should not instantiate non-Cim typ
 
         Import-Clixml -Path (Join-Path $PSScriptRoot "assets\CorruptedCim.clixml")
 
+        # Wait up to 10 seconds for calc.exe to run
```

Comment:
Please use our [`Wait-UntilTrue`](https://github.com/PowerShell/PowerShell/blob/master/test/tools/Modules/HelpersCommon/HelpersCommon.psm1#L1) function.

Reply:
Thats a new function to me, will definitely add that

Reply:
On second reading, may not be appropriate.  WIth other uses of Wait-UntilTrue, we're waiting until something is true, with a 10second time out, checking every second.  Here - we want to wait at least 10 seconds, checking to make sure that calc hasn't started

Maybe just 

```
start-sleep -Seconds 10
Get-Process ... | Should BeNullOrEmpty

?  I'm not as familiar with how a corrupted Cim class could kick off the exe (and have a mac anyway [and the whole thing may not apply in .netcore anyway who knows?]) - I just used the existing Import-Clixml test

Reply:
You can put `Get-Process Calc` to a script block and send it to `Wait-UntilTrue`

Reply:
It seems `Microsoft.Management.Infrastructure` is still not ported https://github.com/PowerShell/PowerShell/issues/4562